### ISSUE TITLE
Cannot find warning source within 2 level include

### ISSUE THREAD

**Issue by dragokas** [Timocop/BasicPawn#116]
Suppose, you have the following code:

test.sp:
```
#pragma semicolon 1
#pragma newdecls required

#include <sourcemod>
#include <L1>

public void OnPluginStart()
{
}
```

L1.inc
```
#include <L2>
```

L2.inc
```
int unused;
```

When you click on this warning:

![image](https://user-images.githubusercontent.com/19956568/129485494-9f64ed6b-3ec6-41c6-8270-ac40ab5dc5b0.png)

the editor doesn't open include file, instead, it points to the current document - test.sp
(however, the line number is correct).

**Comment by Timocop** [comment]
Does this behavior also happen without BasicPawn (running the compiler manually)? If so, you probably want to create an issue in the SourcePawn repo instead since its not BasicPawn related.

**Comment by dragokas** [comment]
Well, you're right. Thanks.

**Comment by dragokas** [comment]
It seems it is already fixed in sm 1.11.
